tox.ini

```diff
@@ -95,6 +95,7 @@ max-line-length = 79
 extend-ignore =
     B024  # abstract class with no abstract methods
     D202
+    D107  # missing docstring in __init__, incompatible with numpydoc
```

Comment:
So is the issue with numpydoc or Sphinx?

Reply:
numpydoc asks for docstings in the class or the init, but not both